src/control/server/mgmt_pool.go

```diff
@@ -397,20 +395,24 @@ func (svc *mgmtSvc) PoolDestroy(ctx context.Context, req *mgmtpb.PoolDestroyReq)
 
 	svc.log.Debugf("MgmtSvc.PoolDestroy dispatch, resp:%+v\n", resp)
 
-	switch drpc.DaosStatus(resp.Status) {
-	case drpc.DaosSuccess:
+	ds := drpc.DaosStatus(resp.Status)
+	switch ds {
+	case drpc.DaosSuccess, drpc.DaosNotLeader, drpc.DaosNotReplica:
+		if ds == drpc.DaosNotLeader || ds == drpc.DaosNotReplica {
```

Comment:
Are these the only responses we might get if it was already destroyed on the data plane side? Any possibility of something like DER_NOTFOUND?

Reply:
I haven't seen other responses. The -DER_NOTREPLICA is most common, where the ranks receive a pool service RPC and respond with that code saying "dunno about that pool, I'm not a replica".